scripts/build_bus_regions.py

```diff
@@ -210,87 +140,50 @@ def get_id(coords):
     for country in countries:
 
         c_b = n.buses.country == country
-        if n.buses.loc[c_b & n.buses.substation_lv, ["x", "y"]].empty:
-            _logger.warning(f"No low voltage buses found for {country}!")
-            continue
 
         onshore_shape = country_shapes[country]
         onshore_locs = n.buses.loc[c_b & n.buses.substation_lv, ["x", "y"]]
-        if snakemake.config["cluster_options"]["alternative_clustering"]:
-            onshore_geometry = get_gadm_shape(onshore_locs, gadm_shapes)[0]
```

Comment:
This deletes the GADM clustering functionalities (see our preprint)
We could create a test for checking `["alternative_clustering"]: True` to detect such breaking changes in future.